tapalcatl_server/storage.go

```diff
@@ -154,3 +165,9 @@ func (f *FileStorage) Fetch(t tapalcatl.TileCoord, c Condition) (*StorageRespons
 		return resp, nil
 	}
 }
+
+func (s *FileStorage) HealthCheck() error {
+	tilepath := filepath.Join(s.baseDir, s.healthcheck)
+	_, err := os.Open(tilepath)
```

Comment:
This is just for testing so not sure how much we should pay attention to it, but in the case when there's no error do we want to close the file?